### PR TITLE
Notify about missing pre-requisites for Wake On Lan

### PR DESCRIPTION
**Pull request by artemkovalyov**
## Proposed change

Mentioning the integrations might make an impression that they are installed by default or are installed together with LG Web OS integration as pre-requisites/dependencies. They are not. Making it explicit helps to avoid unnecessary errors when creating the actions.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **Documentation**
	- Enhanced documentation for the `webostv` integration, including important notes on required integrations and network configurations.
	- Expanded instructions for automations, including example configurations for `wake_on_lan`.
	- Detailed explanations for `play_media` actions and sound output commands, improving clarity on usage and functionality.

### CODE REVIEW

File: source/_integrations/webostv.markdown [hunk 1]
```diff
@@ -34,6 +34,10 @@ To begin with enable *LG Connect Apps* feature in *Network* settings of the TV.
 
 Home Assistant is able to turn on an LG webOS Smart TV if you specify an action, provided by an {% term integration %} like [HDMI-CEC](/integrations/hdmi_cec/) or [WakeOnLan](/integrations/wake_on_lan/).
```
Comment (c0ffeeca7) [inline source/_integrations/webostv.markdown]:
```suggestion
If you want to use an automation to turn on an LG webOS Smart TV, install an {% term integration %} such as the [HDMI-CEC](/integrations/hdmi_cec/) or [WakeOnLan](/integrations/wake_on_lan/). They provide an action that can be used for that.
```

File: source/_integrations/webostv.markdown [hunk 2]
```diff
@@ -34,6 +34,10 @@ To begin with enable *LG Connect Apps* feature in *Network* settings of the TV.
...
 
+{% important %}
+`WakeOnLan` or `HDMI-CED` integrations are not automatically installed together with the `LG webOS Smart TV` integration. Make sure the desired one is already availalbe in `Settings->Devices and services -> Integrations` or use `Add Integration` button to intall it.
+{% endimportant %}
```
Comment (c0ffeeca7) [inline source/_integrations/webostv.markdown]:
```suggestion
```
Maybe the original sentence can be rephrased a bit instead.

File: source/_integrations/webostv.markdown (hunk 1 above)
Comment (artemkovalyov) [reply source/_integrations/webostv.markdown]: Works for me. Although emphasizing and duplicating are not bad things per se when they help making things obvious.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @artemkovalyov :+1:

**Pull request merged**